ocs_ci/utility/templating.py

```diff
@@ -200,3 +201,16 @@ def dump_data_to_temp_yaml(data, temp_yaml):
         yaml_file.write(yaml_data)
     logger.info(yaml_data)
     return yaml_data
+
+
+def dump_data_to_json(data, json_file):
```

Comment:
Is it worth having our own utility that wraps the context manager? It looks like it's only called once.

Reply:
Eventhoug it used once, I have wrapped it for future purpose